### DISCUSSION TITLE
Styling options for inlay hints

### DISCUSSION THREAD

**Discussion by JustLinuxUser** [neovim/neovim#28352] [Ideas]
I really liked what rust-tools.nvim did with their inlay hint implementation. I recognize, that it was probably done due to a limitation of the previous versions of neovim, but I think it is a better option then regular inline hints.

from rust-tools.nvim:
![inlay_hints](https://github.com/neovim/neovim/assets/75628709/39a4b656-57a9-445f-b419-3abd7c4b952a)

Normal inlay hints move text around, which makes it really frustrating to use, and this implementation is much more passive, but still extremely helpful. I would really like a feature like this to be implemented in neovim

**Comment by clason** [comment]
Yes, but are they _inlay_ hints?

Duplicate of https://github.com/neovim/neovim/issues/28261
